app/forms/bank_details_form.rb

```diff
@@ -2,7 +2,7 @@ class BankDetailsForm < Form
   # Only validate against HMRC API if number of attempts is below threshold
   MAX_HMRC_API_VALIDATION_ATTEMPTS = 3
 
-  attribute :hmrc_validation_attempt_count
+  attribute :hmrc_validation_attempt_count, :integer
```

Comment:
I _think_ you can do `attribute :hmrc_validation_attempt_count, :integer, default: 0` to avoid having to overwrite the initialiser (not a blocker)